schedulingApp/templates/addUser.html

```diff
@@ -49,6 +49,10 @@ <h1>Create New User</h1>
                             {% endfor %}
                         </select>
                     </div>
+                    <div class="form-group">

```

Comment:
I would probably remove this from user creation; TAs are adding skills to their own account, and since TAs aren't creating their own accounts, this seems redundant.